lib/serverless/event-sources.json

```diff
@@ -24,13 +25,32 @@
             "requestContext.stage"
         ]
     },
+
+    "apiGatewayV2": {
+        "attributes": {
+            "aws.lambda.eventSource.accountId": "requestContext.accountId",
+            "aws.lambda.eventSource.apiId": "requestContext.apiId",
+            "aws.lambda.eventSource.stage": "requestContext.stage"
+        },
```

Comment:
"resourceId" and "resourcePath", found in the v1 payload, are not present in v2 payloads.